livestream/control.go

```diff
@@ -42,6 +42,9 @@ func (ls *Livestreamer) Start(ctx context.Context, strm Livestream) error {
 			if err != nil {
 				return fmt.Errorf("failed to start yt-existing broadcast: %w", err)
 			}
+
+		case LinkRTMPOutput:
+			return fmt.Errorf("not yet implemented") //FIXME
```

Comment:
This is more a nothing to do case since there isn't anything we can send

(there is some control signals in RTMP but currently showtime doesn't parse the RTMP feed)

With how this is designed won't this block livestreams which have a RTMP output link also?